includes/service-providers/class-newspack-newsletters-service-provider.php

```diff
@@ -187,14 +189,77 @@ public function transition_post_status( $new_status, $old_status, $post ) {
 			return;
 		}
 
+		// Handle scheduled newsletters.
 		if ( in_array( $new_status, self::$controlled_statuses, true ) && 'future' === $old_status ) {
 			update_post_meta( $post->ID, 'sending_scheduled', true );
 			$result = $this->send_newsletter( $post );
 			if ( is_wp_error( $result ) ) {
+				$prior_attempts = intval( get_post_meta( $post->ID, 'scheduled_send_attempts', true ) );
```

Comment:
Why not leverage the existing `newsletter_send_errors` meta for this? It's persisted and each error entry is already an attempt.

Reply:
That's a good idea!

Reply:
f430275 changes the approach to use the `newsletter_send_errors` meta.